transcom/mymove
# B 20990 main - Add Counseling Office Column and Filtering to TOO Queue

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20990)

## Summary
When a customer selects an origin duty location assigned to government counseling, then they will also be asked to select a counseling office within the same GBLOC. This field will be called "Counseling Office".

### How to test
1. Login as a customer and create a 3 moves all with different counseling offices.
2. Login as a  Services Counselor and submit the new move.
3.  Login as TOO,  verify the Counseling Office Column is displayed In the task order queue.
4. Filter the  Counseling Office Task Order Queue for a move to verify filtering works.
5. Sort the Counseling Office in the Task Ordering Queue to verify sort works. 

## [INT PR](https://github.com/transcom/mymove/pull/14142)
